pubsub/listener.rb

```diff
@@ -0,0 +1,32 @@
+# Copyright 2015 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "sinatra"
+require "json"
+require "base64"
+
+configure do
+  set :port, 8080
+end
+
+get "/_ah/health" do
```

Comment:
Health check not required.  Requests to `/_ah/health` simply need to respond with a non 5xx status code.  Sinatra will return a 404 by default which is a valid health check response.